source/extensions/transport_sockets/alts/BUILD

```diff
@@ -0,0 +1,86 @@
+licenses(["notice"])  # Apache 2
```

Comment:
Also needs CODEOWNERS update per the new extension policy.  Your call who you pull in for your second - I'm up for it but you can also tag someone else from Istio or ALTS team if you think that makes more sense.

Reply:
The new extension policy doesn't allow senior maintainers to be in CODEOWNERS :/

> Each extension must have two reviewers proposed for reviewing PRs to the extension. Neither of the reviewers must be a senior maintainer.